### PR TITLE
add webhook step to drone build to update ocis.owncloud.works

### PR DESCRIPTION
**Pull request by C0rby**
Before this gets merged we need to add the `webhook_username` and `webhook_password` to the secrets.

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -612,6 +613,42 @@ def binary(ctx, name):
...
+          },
+          'password': {
+            'from_secret': 'webhook_password',
+          },
+          'method': 'GET',
+          'urls': 'https://ocis.owncloud.works/hooks/update-ocis',
+        }
+      }
+    ],
+    'depends_on': [
```
Comment (micbar) [inline .drone.star:641]: please add more deps here.
We need to trigger only if all docker pipelines were successful

File: .drone.star [hunk 2]
```diff
@@ -71,6 +71,7 @@ def main(ctx):
   ]
 
   after = [
+    updateDeployment(ctx)
```
Comment (micbar) [inline .drone.star:74]: move to the last line

File: .drone.star (hunk 1 above)
Comment (C0rby) [reply .drone.star:641]: Is that so? Because our instance is an amd64 Linux machine. But sure it doesn't hurt to have all other steps finish first.

Comment (micbar) [reply .drone.star:641]: yes, correct, but we should't deploy if we have errrors on other platforms. We should investiate.
Agreed?

Comment (C0rby) [reply .drone.star:641]: True. :+1:

**Review by micbar — APPROVED** [review]
looking good now.
We will only know if it works after the merge 😄

**Comment by C0rby** [comment]
Yeah, hopefully it does work. :D
I want to move on to other tasks. So much to do. :see_no_evil:

**Pull request merged**